package.json

```diff
@@ -57,6 +57,7 @@
     "rollup-plugin-uglify": "^1.0.1",
     "rollup-plugin-vue": "^2.3.1",
     "rollup-watch": "^3.2.2",
+    "vue": "2.2.5",
```

Comment:
Is there a reason to have vue in package.json twice and different versions?
I had npm complains about it.